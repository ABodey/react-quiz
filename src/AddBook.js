import React, { Fragment, PureComponent } from 'react'
import PropTypes from 'prop-types'

class AddBook extends PureComponent {
    static propTypes = {
        onAddBook: PropTypes.func.isRequired,
    }
    render() {
        const { onAddBook } = this.props

        return (
            <form
                autoComplete="off"
                onSubmit={onAddBook}
            >
                <input
                    type="text"
                    name="title"
                    placeholder="Title"
                    required
                />
                <input
                    type="text"
                    name="author"
                    placeholder="Author"
                    required
                />
                <button type="submit">Add Book</button>
            </form>
        )
    }

}

export default AddBook