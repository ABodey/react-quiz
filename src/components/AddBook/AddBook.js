import React from 'react'

const AddBook = () => (
  <h1> AddBook </h1>
)

export default AddBook
