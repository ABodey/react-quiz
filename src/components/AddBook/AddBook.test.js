import React from 'react'
import { shallow } from 'enzyme'
import AddBook from './AddBook'

describe('<AddBook />', () => {
    it ('should match snapshots', () => {
      const wrapper = shallow (<AddBook />)

      expect(wrapper).toMatchSnapshot()
    })
})

