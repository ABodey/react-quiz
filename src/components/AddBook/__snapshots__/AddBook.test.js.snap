// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddBook /> should match snapshots 1`] = `
<h1>
   AddBook 
</h1>
`;
